ADR/Components/Aerodynamic_components/tests/test_HS.py

```diff
@@ -0,0 +1,46 @@
+from ADR.Components.Aerodynamic_components.HS import HS
+import numpy.testing as npt
+import numpy as np
+import pytest
+
+
+@pytest.fixture
+def horizontal_stabilizer():
+    aerodynamic_surface_parameters = {
+        "airfoil_clmax": 2.5,
+        "airfoil1_name": "s1223",
+        "airfoil2_name": "s1223",
+        "airfoil3_name": "s1223",
+        "span1": 2,
+        "span2": 1.5,
+        "chord1": 0.40,
+        "chord2": 0.30,
+        "chord3": 0.35,
+        "twist1": 0,
+        "twist2": 0,
+        "twist3": 0,
+        "incidence": 0,
+        "x": 1,
+        "z": 1,
+    }
+    horizontal_stabilizer = HS(aerodynamic_surface_parameters)
```

Comment:
Much better!

Just the aerodynamic_surface_parameters variable is missing renaming.